internal/cli/atlas/backup/compliancepolicy/policies/scheduled/update.go

```diff
@@ -1,116 +0,0 @@
-// Copyright 2023 MongoDB Inc
```

Comment:
this is atlas command, does it have to be removed?

Reply:
this file is unused looking at https://www.mongodb.com/docs/atlas/cli/stable/command/atlas-backups-compliancePolicy-policies-scheduled-create/ looks like update is not present

Reply:
also it looks commented at https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/cli/atlas/backup/compliancepolicy/policies/scheduled/scheduled.go#L39